spec/listen/array-changes-spec.js

```diff
@@ -46,7 +46,7 @@ describe("Array change dispatch", function () {
         // are both non-enumerable, and any other properties that might get
         // added in the future.
         for (var name in array) {
-            expect(isNaN(+name)).toBe(false);
+            expect(isNaN(+name)).toBeFalsy();
```

Comment:
Not sure what this relaxation achieves. Falsy is less strict than False. But while I’m looking at it, `isNaN` coerces its argument to Number, so the `+` is superfluous. `x !== x` is the way to check for NaN without risk of coercion.